### PR TITLE
stackSubgroups option

### PR DESCRIPTION
**Pull request by yotamberk**
Allow doing doing things like: #497 (see subgroups example)

### CODE REVIEW

**Comment by mojoaxel** [comment]
@yotamberk Can you please create a new example for this new feature and lead the old subgroup example as it is.

**Comment by yotamberk** [comment]
sure

**Comment by yotamberk** [comment]
@mojoaxel done

File: examples/timeline/groups/stackSubgroup.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        {
+            id: 'group2', 
+            content:'group2', 
+            subgroupOrder: 'subgroupOrder' // this group has no subgroups but this would be the other method to do the sorting.
+        }
+    ]);
+
+    // add items to the DataSet
+    items.add([
+    // group 1
```
Comment (mojoaxel) [inline examples/timeline/groups/stackSubgroup.html]: indent

File: examples/timeline/groups/stackSubgroup.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        },
+         {
+            id: 6, 
+            content: 'item 3 (actual time)', 
+            start: '2014-02-05 12:00:00', 
+            end: '2014-02-08 12:00:00',
+            group:'group1', 
+            subgroup:'sg_3'
+        },
+    // group 2
```
Comment (mojoaxel) [inline examples/timeline/groups/stackSubgroup.html]: indent

File: examples/timeline/groups/stackSubgroup.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        // items without subgroups
+        {
+            id: 0, 
+            content: 'no subgroup', 
+            start: '2014-01-20', 
+            end: '2014-01-22',
+            group:'group2'
+        },
+    // general
```
Comment (mojoaxel) [inline examples/timeline/groups/stackSubgroup.html]: indent

File: examples/timeline/groups/stackSubgroup.html [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+        .vis-item.vis-background.marker {
+            border-left: 2px solid green;
+        }
+    </style>
+    <script src="../../../dist/vis.js"></script>
+    <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+<p>This example shows the workings of the subgroups. Subgroups do not use stacking, and only work when stacking is disabled.
```
Comment (mojoaxel) [inline examples/timeline/groups/stackSubgroup.html]: As far as I can see it this example shows two new concepts:
* Make subGroups stackable
* show overlay of elements like descripted in #497

I would recommend integrating the "stackSubgroups" option in the existing "subgroups" example and creating a new example that only shows how to realize the #497 feature. I think this are two separated thinks.

File: examples/timeline/groups/stackSubgroup.html [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+        },
+         {
+            id: 4, 
+            content: 'item 2 (actual time)', 
+            start: '2014-01-14 12:00:00', 
+            end: '2014-01-17 12:00:00',
+            group:'group1', 
+            subgroup:'sg_2'
+        },
+        // subgroup 3
```
Comment (mojoaxel) [inline examples/timeline/groups/stackSubgroup.html]: maybe 2 subgroups are enough? The examples should be as "slim" as possible.

File: examples/timeline/groups/stackSubgroup.html (hunk 4 above)
Comment (yotamberk) [inline examples/timeline/groups/stackSubgroup.html]: will do

**Comment by yotamberk** [comment]
@mojoaxel 
>As far as I can see it this example shows two new concepts:
Make subGroups stackable
show overlay of elements like descripted in #497
I would recommend integrating the "stackSubgroups" option in the existing "subgroups" example and creating a new example that only shows how to realize the #497 feature. I think this are two separated thinks.

I liked that comment. I did it.

**Pull request merged**